Tests/Unit/NetworkServer/LoRaDeviceTest.cs

```diff
@@ -27,14 +28,46 @@ namespace LoRaWan.Tests.Unit.NetworkServer
     /// </summary>
     public class LoRaDeviceTest
     {
+        private static readonly NetworkServerConfiguration Configuration = new NetworkServerConfiguration { GatewayID = "test-gateway" };
+        private static readonly LoRaDesiredTwinProperties OtaaDesiredTwinProperties = new LoRaDesiredTwinProperties
```

Comment:
Perhaps we can skip saying `LoRaDesiredTwinProperties` twice since it doesn't add much value, here and elsewhere below?

```suggestion
        private static readonly LoRaDesiredTwinProperties OtaaDesiredTwinProperties = new
```

Reply:
Applied for this occurrence and the other `static readonly` occurrences in the same test as part of 0bc8919c90a7ce0c10880fea37c72d4f354b5b4d